src/app/components/cohort-view/cohort-view.directive.js

```diff
@@ -97,6 +99,27 @@ angular.module('transmartBaseUi')
                     });
                 };
 
+
+                ctrl.loadCohort = function(cohort) {
+
+                    // Get the controller for the selected workspace
+                    var cohortSelectionCtrl = CohortSelectionService.boxes[0].ctrl;
```

Comment:
we should load it to the current box, the one that the user has created most recently.